src/core/languages/csharp.ts

```
@@ -0,0 +1,18 @@
+import type { LanguageConfig } from './index.js';
+
+export const CSharpConfig: LanguageConfig = {
+  name: 'C#',
+  extensions: ['.cs'],
+
+  loadGrammar: async () => {
+    const CSharpLanguage = await import('tree-sitter-c-sharp');
+    return CSharpLanguage.default;
+  },
+
+  extractSymbols: (tree: Parser.Tree, sourceCode: string): Symbol[] => {
```

ESLint

[error] 'tree' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)